src/views/Clients/ViewClient/index.jsx

```diff
@@ -23,15 +33,43 @@ import clientQuery from './client.graphql';
     },
   }),
 })
+@withStyles(theme => ({
+  listItemButton: {
+    ...theme.mixins.listItemButton,
+  },
```

Comment:
Since the background should be yellow, the colors will be inverted since now text need to be somewhat black.

```jsx
listItemButton: {
  padding: 0,
  '& svg': {
    transition: transitions.create('fill'),
    fill: fade(
      theme.palette.type === 'light' ? THEME.PRIMARY_TEXT_LIGHT : THEME.PRIMARY_TEXT_LIGHT,
      0.4
    ),
  },
  '&:hover svg': {
    fill: theme.palette.type === 'light' ? THEME.PRIMARY_TEXT_DARK : THEME.PRIMARY_TEXT_LIGHT,
  },
},
```

Reply:
Just to clarify, in the line `fill: fade(
theme.palette.type === 'light' ? THEME.PRIMARY_TEXT_LIGHT: THEME.PRIMARY_TEXT_LIGHT,
      0.4
    ),`, you mean `'light? THEME.PRIMARY_TEXT_LIGHT: THEME.PRIMARY_TEXT_DARK` right?

Reply:
The background of the card will always be yellow so I think you can simply do `THEME.PRIMARY_TEXT_LIGHT` to make sure the effect stays the same on both themes:

```jsx
listItemButton: {
  padding: 0,
  '& svg': {
    transition: transitions.create('fill'),
    fill: fade(
      THEME.PRIMARY_TEXT_LIGHT,
      0.4
    ),
  },
  '&:hover svg': {
    fill: THEME.PRIMARY_TEXT_LIGHT,
  },
},
```